### PR TITLE
Update cardano-key-pairs.md

### PR DESCRIPTION
**Pull request by D3vJohn**
Improved the language.

### CODE REVIEW

**Review by katomm — CHANGES_REQUESTED** [review]
Thank you for taking the time. Unfortunately, a few important tags were deleted. (single backticks for the codeblocks)

`keyname.vkey` vs 'keyname.vkey' vs 'keyname.vkey,'. Could you look into this please?

File: docs/operate-a-stake-pool/cardano-key-pairs.md [hunk 1]
```diff
@@ -6,21 +6,20 @@ description: Learn about Cardano key pairs.
...
+Cardano cryptographic keys are made up of 'ed25519' key pairs, which include a 'public verification key file' and a'secret (private) key file.' The public key file is commonly referred to as 'keyname.vkey,' whereas the private key file is referred to as 'keyname.skey.' The private key file, which is used to sign transactions, is extremely sensitive and should be adequately safeguarded. Under all circumstances, this entails limiting third-party access to your private keys. The most effective technique to prevent private key exposure is to guarantee that the necessary private key is never held for any length of time on any internet-connected machine (hot node). Please note that key pair filenames are completely random and can be named whatever you want.
```
Comment (rdlrt) [inline docs/operate-a-stake-pool/cardano-key-pairs.md:11]: the use of tilde sign helps visibility on the rendered page, compare to single quotes, other than that - all good

**Review by D3vJohn — COMMENTED** [review]
Added code tags in the appropriate places.

**Review by katomm — APPROVED** [review]
Thank you!




[15:25](https://cardano-ambassadors.slack.com/archives/D02K1MDS7ML/p1646835935281389)
Next translator call is March 16





[15:26](https://cardano-ambassadors.slack.com/archives/D02K1MDS7ML/p1646835960084149)
Oh, I think this email was fired because the time slightly changed on that date because of a clash

**Pull request merged**